addons/website_forum/static/src/js/website_forum.js

```diff
@@ -253,9 +254,8 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             // translation, to fix in the appropriate version
             notifOptions.message = `${karma} ${_t("karma is required to perform this action. ")}`;
             if (forumID) {
-                notifOptions.messageIsHtml = true;
                 const linkLabel = _.escape(_t("Read the guidelines to know how to gain karma."));
```

Comment:
This here should not be escaped explicitely. The entire point of `Markup` as template tag is that it escapes interpolated values automatically. 

`_.escape` does not generate markup objects, as that was a compatibility issues (because in js `String` subclasses are not fully substitutable with unboxed `string` values).